libs/core/src/Core.zig

```diff
@@ -7,8 +7,10 @@ pub const Core = @This();
 
 internal: platform.Core,
 
+pub const AppType = enum { Standard, Game, Headless };
+
 pub const Options = struct {
-    is_app: bool = false,
+    app_type: AppType = .Game,
```

Comment:
I suspect the solution here will be to:

1. Keep `is_app: bool = false,`
2. Additionally add `headless: bool = false,` (below `size` field) which we can respect at `init`

Reply:
Sure, that works as well

Reply:
cool, if you're happy with that I'm happy

want to send a new PR or update this one? or prefer if I make that change? either way is fine with me